tests/blocks.cairo

```diff
@@ -1,8 +1,13 @@
 pub mod block_0;
-pub mod block_170;
```

Comment:
Were there any issues with this particular block?

Reply:
Not particularly, I just forgot to take it back (he is now in the blocks). But for the block 481824 i don't think it was optimal to save a 6Mo test file in the git.